docs/DeveloperGuide.md

```diff
@@ -703,39 +700,198 @@ testers are expected to do more *exploratory* testing.
-      Expected: Similar to previous.
+1. Prerequisites: 
+   1. Arguments must be specified.
+   1. Module to be included must exist in the list being displayed in GradPad, e.g. CS2100, CS2101, CS3230 and ST2334 are in the list and CS2106 is not.
+      
+1. Test case: `find cs2`<br>
+   Expected: CS2100 and CS2101 are displayed.
+   
+1. Test case: `find cs2 st`<br>
+   Expected: CS2100, CS2101 and ST2334 are displayed.
+   
+1. Test case: `find cs3230`<br>
+   Expected: CS3230 is displayed.
+
+1. Test case: `delete cs2106`<br>
+   Expected: No modules displayed.
+      
+1. Test case: `find`<br>
+   Expected: Current list is unchanged. _Invalid command format_ message is shown in the result display.
+   
+### Check Total Modular Credits
+
+1. Prerequisite: Command must not be accompanied by any arguments.
+
+1. Test case: `checkmc`
+   Expected: Total modular credits are calculated and displayed. If there are no modules in 'Completed Modules', total modular credits will be 0.
+   
+1. Test case: `checkmc modules`
+   Expected: Total modular credits are not calculated. _Invalid command format_ message is shown in the result display.
+   
+### Open Help Page
+
+1. Prerequisite: Command must not be accompanied by any arguments.
+
+1. Test case: `help`
+   Expected: Help page is displayed.
+   
+1. Test case: `checkmc modules`
+   Expected: Help page is not displayed. _Invalid command format_ message is shown in the result display.
+   
+### Show Required Modules
+
+1. Prerequisite: Command must not be accompanied by any arguments.
+
+1. Test case: `required`
+   Expected: All required modules are displayed in the result display. Modules already in the 'Completed Modules' list in GradPad would not be displayed in the 'Required Modules' list.
+   
+1. Test case: `required modules`
+   Expected: Required modules are not displayed. _Invalid command format_ message is shown in the result display.
+   
+### Check Module Information
+
+1. Prerequisites: 
+   1. Module code must be specified.
+   1. Module code format must be valid.
+   1. Module to be searched must exist in the valid modules list fetched from NUSMods, e.g. module code CS2100 is a valid module, whereas module code CS1000 is an invalid module.
+ 
+1. Test case: `search cs2100`<br>
+   Expected: CS2100 module information is displayed in the result display.
+   
+1. Test case: `search cs2100 Computer Ogranisation`<br>
+   Expected: No module information is displayed. _Invalid module code format_ message is shown in the result display.
+
+1. Test case: `search cs1000`<br>
+   Expected: No module information is displayed. _Invalid module_ message is shown in the result display.
+      
+1. Test case: `search`<br>
+   Expected: No module information is displayed. _Invalid command format_ message is shown in the result display.
 
-1. _{ more test cases …​ }_
+1. Other invalid delete commands to try: `search c/cs2103t`, `delete 1`<br>
+   Expected: Similar to previous.
 
 ### Saving data
```

Comment:
think we can remove this line haha

Reply:
HAHAHAHA bro i just followed the given, not sure if correct syntax :') maybe i should just put the same line as the one above instead of putting "Similar to previous." 🤣

Reply:
Ya HAHAH i think that would be a lot clearer